Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/setup.py

```diff
@@ -301,12 +301,28 @@ def parser(cls, parser, loggers=None):
 
     @classmethod
     def main(cls, args, repository, **kwargs):
-        print('For detailed information about options this script is configuring see:')
-        print('https://github.com/WebKit/WebKit/wiki/Git-Config#Configuration-Options')
-
         if isinstance(repository, local.Git):
+            if 'true' != repository.config().get('webkitscmpy.setup', ''):
```

Comment:
should it be called `webkitscmpy.setup` or something like `git-webkit.setup`?

Reply:
I've been prefixing with `webkitscmpy` since that's the layer that actually owns these names. It's not our first git config option: https://github.com/WebKit/WebKit/blob/main/metadata/project_config